sdk/reya_websocket/resources/market.py

```
@@ -266,53 +269,28 @@ def unsubscribe(self) -> None:
         self.socket.send_unsubscribe(channel=self.path)
 
 
-class MarketSpotExecutionBustsResource(SubscribableParameterizedResource):
-    """Resource for accessing market spot execution busts."""
+class MarketExecutionBustsResource(SubscribableParameterizedResource):
+    """Resource for accessing market execution busts (unified spot + perp)."""
 
     def __init__(self, socket: "ReyaSocket"):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)